### DISCUSSION TITLE
How to change the e-mail for receiving notifications

### DISCUSSION THREAD

**Discussion by ShadyMedic** [community/community#33959] [New to GitHub] [answered]
Heya, I'm not exactly new to GitHub, but I can't just solve this on my own.

I am migrating from Gmail to Posteo for security reasons and I'm in the process of changing my e-mail address everywhere I created an account. GitHub included.

When I went to my e-mail settings, I could easily add my Posteo e-mail and change it into the primary one. However, I still receive notification e-mails about my pull requests etc. into my old Gmail inbox.

This is what I see:
![image](https://user-images.githubusercontent.com/19894267/191995516-67cdd6b2-b5a4-41fe-a364-90397b59a891.png)

How can I move the `Receives notifications` information to the now primary e-mail address?

NOTE: I don't want to remove the Gmail address, as it has thousands of my commits associated with it.

**Comment by airtower-luna** [comment, marked as answer]
Check the "Notifications" part of your settings (in the menu on the left), you can choose the address for notifications there.

**Reply by ShadyMedic** [reply]
Great! Thank you.

**Comment by Justinzobel** [comment]
This should be in the Emails section or at least a note with a link from Emails to the Notifications page.

**Comment by hhaddougui** [comment]
It is in the last main tab of your repository called settings. There you head over to email notifications option under Integrations, and you start adding up to two email adresses to receive notification once there is a push to your repo or pull request or something different.
Did someone observed that github policies related to this setting don't disallow adding identical duplicate adresses with same value without returning error ?
